shared/logutil/logutil.go

```diff
@@ -5,10 +5,13 @@ package logutil
 import (
 	"io"
 	"os"
-
+	"time"
 	"github.com/sirupsen/logrus"
+	"github.com/prysmaticlabs/prysm/shared/roughtime"
 )
 
+var log = logrus.WithField("prefix", "forkchoice_checker")
```

Comment:
Please update this. Forkchoice checker is not an appropriate name.

Reply:
I'll change it to `logutil`